lnwire/update_fee.go

```diff
@@ -0,0 +1,63 @@
+package lnwire
+
+import (
+	"io"
+
+	"github.com/roasbeef/btcutil"
+)
+
+// UpdateFee is the message the channel initiator sends to the other peer if
+// the channel commitment fee needs to be updated.
+type UpdateFee struct {
+	ChanID ChannelID
+
+	FeePerKw btcutil.Amount
+}
+
+// NewUpdateFee creates a new UpdateFee message.
```

Comment:
The test in `lnwire_test.go` should be updated to include a scenario for this new message.